src/Core/maps/src/Handlers/Map/MapHandler.Android.cs

```diff
@@ -71,6 +67,9 @@ protected override void DisconnectHandler(MapView platformView)
 
 		protected override MapView CreatePlatformView()
 		{
+			// Make sure to use the latest renderer, must be called before creating the MapView
+			MapsInitializer.Initialize(Context, MapsInitializer.Renderer.Latest, _mapReady);
```

Comment:
Should this check before if Google Play Services are available?

Reply:
That is already checked when you enabled Maps to begin with:

https://github.com/dotnet/maui/blob/main/src/Controls/Maps/src/AppHostBuilderExtensions.cs#L44

I guess I could move this call all the way to up there to make sure that we really call this before any Map is created?

Reply:
Actually I did move it to the UseMauiMaps call, that seems like a better place.